kafka/consumer.go

```diff
@@ -450,10 +476,15 @@ func (c *consumer) processFetch(ctx context.Context, fetches kgo.Fetches) {
 			}
 			return
 		}
+		done := make(chan struct{})
+		go func() {
+			defer close(done)
+			consumer.consumeRecords(ftp)
+		}()
 		var err error
 		select {
 		// Send partition records to be processed by its dedicated goroutine.
```

Comment:
I wonder if "dedicated goroutine" still applies here. They are not goroutines readily waiting for records now.

Reply:
Good catch, I'll update the comment.

Reply:
Fixed in the last commit.